sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/WebProxyConverter.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Amqp.Config
+{
+    internal class WebProxyConverter : JsonConverter<IWebProxy>
+    {
+        public override void WriteJson(JsonWriter writer, IWebProxy value, JsonSerializer serializer)
+        {
+            writer.WriteValue(value.GetProxy(new Uri("https://servicebus.windows.net")));
```

Comment:
What if this URL pattern is not accurate? Such as when running on Azure Stack? Should we consider using the actual endpoint?